api/controllers/game/join-rematch.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Join the two players into the new game
+ */
+module.exports = async function (req, res) {
+  try {
+    const { usr: userId, game: gameId } = req.session;
+    const user = await User.findOne({ id: userId });
+    const oldGame = await Game.findOne({ id: gameId });
```

Comment:
These two could be done in parallel
```suggestion
    const [ user, oldGame ] = await Promise.all([
      User.findOne({ id: userId }),
      Game.findOne({ id: gameId })
    ]);
```